classifier_load_testing/create_groups.rb

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+
+require_relative './helpers'
+
+num_nodes = get_arg("num_nodes", 0)
+
+env_group_parent_id = get_group_id('all-snow-environments')
+cv_group_parent_id = get_group_id('all-snow-classes-vars')
+
+num_nodes.to_i.times do |node|
```

Comment:
Note that this works. I see `node0`, `node1` groups under both `all-snow-environments` and `all-snow-classes-vars`.